packages/lib-classifier/src/plugins/tasks/components/TaskInput/components/TaskInputLabel/TaskInputLabel.js

```diff
@@ -15,6 +15,7 @@ const StyledTaskInputLabelWrapper = styled.span`
 `
 
 const StyledText = styled(Text)`
+  align-content: center;
   display: block;
```

Comment:
This is `display: block` not flex, so `align-content` won't apply. Did you mean to include this change?

Reply:
So, when I revert this the alignment looks off. My understanding is that the `align-content` helps with the current element in the context of it's parent `flex` attribute. Could be wrong, but was trying to fix the visual issue you see in the screenshots below. Happy to revert if it should be handled in another way.

![missing-align-content](https://github.com/user-attachments/assets/8dee3b34-8ecf-4
![with-align-content](https://github.com/user-attachments/assets/44a0b48c-4996-4e65-a202-cb5e4b143784)
da5-9165-166432b02712)

Reply:
Ah I see. Through PR history it looks like the `display: block` was copy pasted from when the label text was a `span` instead of a Grommet Text. This alignment change is fine to keep in this PR, but I recommend removing `display: block` to avoid confusion during code read. TBH it doesn't even need the `margin: 1em 0` either. That was probably an attempt to vertically align the text awhile back, now solved by `align-content` instead.